### DISCUSSION TITLE
Long text automatic left indent

### DISCUSSION THREAD

**Discussion by steve-alpoz** [freeplane/freeplane#590] [Help] [answered]
Is there a setting to change or remove the long text left indent for a node?
![image](https://user-images.githubusercontent.com/66113818/186566307-e51cc14d-d321-4d2b-aedf-b0c9f3c29390.png)
I know I can change the max node width but that's not the solution I'm after.

**Comment by macmarrum** [comment]
Try removing formatting of the text (maybe it was copied/pasted as rich text and the indent was included):
* `Edit->Node core->Edit node core in dialog (Alt+Enter)`
  * `Format->Remove formatting (Ctrl+T)`

OR
* `Format->Remove format`

If that doesn't solve it, please specify which version of Freeplane you are using, attach your sample mindmap and your `auto.properties`. You'll need to zip the files first.

You'll find your `auto.properties` file in `Tools->Open user directory`.

**Comment by steve-alpoz** [comment]
Thanks.  Neither of the format removing suggestions worked.

I've zipped the auto.properties and a sample mind which I just newly created. It is exhibiting the same behaviour. 

I'm running v 1.10.3 on Win 10. Screenshot included in the zip. This has been happening for me for a long time though.

In case it's relevant, the only add-on's I'm running are Jumper 1.0.1 an FreeplaneGTD 0.8.1.

[auto-indent-test.zip](https://github.com/freeplane/freeplane/files/9422355/auto-indent-test.zip)

**Reply by macmarrum** [reply]
> In case it's relevant, the only add-on's I'm running are Jumper 1.0.1 an FreeplaneGTD 0.8.1.

A good hunch. It's FreeplaneGTD 0.8.1 that messes it up.

My way to fix it: 
* `Tools->Open user directory` 
* Go to `addons/freeplaneGTD/scripts/` 
* Edit `freeplaneGTD.groovy` 
* Find `padding: 10px 25px 0px 25px;` and replace with `/*padding: 10px 25px 0px 25px;*/`
* Save (and close) the file
* Restart Freeplane
* Run FreeplaneGTD->Next action list

Alternatively, add CSS to `Default` style in your map(s) to revert CSS set by FreeplaneGTD
```css
body { padding: 0 0 0 0; }
```
See also https://docs.freeplane.org/#/user-documentation/css-for-html-or-markdown

**Comment by steve-alpoz** [comment, marked as answer]
**That solved it!** Thanks.

I edited the freeplaneGTD.groovy file as suggested.

Thanks also for the enlightenment on adding CSS. I can see potential uses for that in future projects.
